### PR TITLE
#3 Create Competency Model

### PR DESCRIPTION
**Pull request by rahuang**
#3 
  - Created Competency Model
  - Created Unit tests (100% model coverage)
    - Created sets and contexts for competencies
    - Created factories
  - Updated Gemfile

### CODE REVIEW

**Comment by tchambers3** [comment]
Did you comment out the has_many:indicators because you weren't sure if indicators could belong to multiple competencies?

**Comment by rahuang** [comment]
So the commented out relationship is more for when I create an indicator later on. I just kept it there for now

File: app/models/competency.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Competency < ActiveRecord::Base
+  # Relationships
+  # has_many :indicators
```
Comment (tchambers3) [inline app/models/competency.rb:3]: I was talking about this comment

Comment (rahuang) [reply app/models/competency.rb:3]: Yea im basically keeping it there for future use of it when I create the indicator model

File: app/models/competency.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  # Validations
+  validates_presence_of :name
+
```
Comment (ppuri96) [inline app/models/competency.rb]: should you also validate the presence of description? Kim made it seem like all competencies will come with short descriptions.

**Pull request merged**